compute_sdk/tests/unit/test_authorizer_login_manager.py

```diff
@@ -21,17 +36,32 @@ def test_auth_client_requires_authorizer_openid_scope(mocker):
     alm = AuthorizerLoginManager({})
     with pytest.raises(KeyError) as pyt_exc:
         alm.get_auth_client()
-    assert AuthScopes.openid in str(pyt_exc)
+    assert AuthScopes.resource_server in str(pyt_exc)
+
+
+def test_get_auth_client(mock_authorizers: dict[str, RefreshTokenAuthorizer]):
+    alm = AuthorizerLoginManager(mock_authorizers)
+    auth_client = alm.get_auth_client()
+    assert isinstance(auth_client, AuthClient)
+    assert auth_client.authorizer == mock_authorizers[AuthScopes.resource_server]
+
+
+def test_get_web_client(mock_authorizers: dict[str, RefreshTokenAuthorizer]):
+    alm = AuthorizerLoginManager(mock_authorizers)
+    web_client = alm.get_web_client()
+    assert isinstance(web_client, WebClient)
+    assert web_client.authorizer == mock_authorizers[ComputeScopes.resource_server]
 
 
-def test_does_not_open_local_cred_storage(mocker, randomstring):
-    test_authorizer = randomstring()
+def test_does_not_open_local_cred_storage(
+    mocker, randomstring, mock_authorizers: dict[str, RefreshTokenAuthorizer]
+):
     mock_lm = mocker.patch(f"{_MOCK_BASE}authorizer_login_manager.LoginManager")
-    mock_lm.SCOPES = {test_authorizer}
+    mock_lm.SCOPES = {key: [randomstring] for key in mock_authorizers.keys()}
     with pytest.raises(LookupError):
```

Comment:
Friendly reminder that `randomstring` is a callable _to get_ a random string.  Doh!

Reply:
Nice catch. I just pushed an update.